### PR TITLE
fix(locale): add missing cs_CZ entry

### PR DESCRIPTION
**Pull request by antleblanc**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-5648
| License          | BSD 3-Clause

#### :bug: Bug Fixes

d9be19c - fix(locale): add missing cs_CZ entry
293b588 - fix: export localeStorageKey constant as default
5cdf0c2 - fix(environment.service): temporarily fallback for cs_CZ language

#### :bug: Bug Fixes

ddc504e - fix(language.menu): temporarily force display of cs_CZ language

### :link: Relates

#3795

### CODE REVIEW

**Review by lizardK — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report test#15894.0 ✘  
* Stage 1
  * lint ✔ 
  * SemVer ✘

**Review by cbourgois — DISMISSED** [review]

File: packages/components/ng-ovh-payment-method/package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "@ovh-ux/component-rollup-config": "^7.0.0"
   },
   "peerDependencies": {
-    "@ovh-ux/manager-config": "^1.0.0 || ^2.0.0",
+    "@ovh-ux/manager-config": "^0.4.0 || ^1.0.0",
```
Comment (cbourgois) [inline packages/components/ng-ovh-payment-method/package.json:47]:
```suggestion
    "@ovh-ux/manager-config": "^1.0.0 || ^2.0.0",
```
Maybe the revert should not revert package version...
Current `@ovh-ux/manager-config` version is `1.1.1` (https://github.com/ovh/manager/blob/develop/packages/manager/modules/config/package.json#L3) but will be updated to `2.0.0` in the next release with https://github.com/ovh/manager/commit/014d810e21f79b0ebf50d0aaf133328d37c4ba4b.

Comment (antleblanc) [reply packages/components/ng-ovh-payment-method/package.json]: As discussed, these reverts can be omitted due to the fact that a fix is also required regarding this Pull Request.

**Review by marie-j — APPROVED** [review]
**thought**: Maybe available languages could be feature flipped ?

**Comment by antleblanc** [comment]
> **thought**: Maybe available languages could be feature flipped ?

Totally agree. Planned for following ticket MANAGER-5646.

**Review by cbourgois — APPROVED** [review]

**Review by cbourgois — DISMISSED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by Jisay — DISMISSED** [review]

**Review by marie-j — APPROVED** [review]

**Pull request merged**